inception-external-search-solr/src/main/java/de/tudarmstadt/ukp/inception/externalsearch/solr/traits/SolrSearchProviderTraitsEditor.html

```diff
@@ -0,0 +1,96 @@
+        <div class="form-group form-row" wicket:enclosure="remoteUrl">
+            <label class="col-sm-2 col-form-label">
+                <wicket:message key="remoteUrl"/>
+            </label>
+            <div class="col-sm-10">
+                <input wicket:id="remoteUrl" type="text" class="form-control"/>
+            </div>
+        </div>
+        <div class="form-group form-row" wicket:enclosure="indexName">
+            <label class="col-sm-2 col-form-label">
+                <wicket:message key="indexName"/>
+            </label>
+            <div class="col-sm-10">
+                <input wicket:id="indexName" type="text" class="form-control"/>
+            </div>
+        </div>
+
+        <!-- DELETE ? -->
+
+        <div class="form-group form-row" wicket:enclosure="searchPath">
+            <label class="col-sm-2 col-form-label">
+                <wicket:message key="searchPath"/>
+            </label>
+            <div class="col-sm-10">
+                <input wicket:id="searchPath" type="text" class="form-control"/>
+            </div>
+        </div>
+
+
+        <div class="form-group form-row" wicket:enclosure="defaultField">
+            <label class="col-sm-2 col-form-label">
+                <wicket:message key="defaultField"/>
+            </label>
+            <div class="col-sm-10">
+                <input wicket:id="defaultField" type="text" class="form-control"/>
+            </div>
+        </div>
+        <div class="form-group form-row" wicket:enclosure="textField">
+            <label class="col-sm-2 col-form-label">
+                <wicket:message key="textField"/>
+            </label>
+            <div class="col-sm-10">
+                <input wicket:id="textField" type="text" class="form-control"/>
+            </div>
+        </div>
+        <div class="form-group form-row" wicket:enclosure="resultSize">
+            <label class="col-sm-2 col-form-label">
+                <wicket:message key="resultSize"/>
+            </label>
+            <div class="col-sm-10">
+                <input wicket:id="resultSize" type="number" class="form-control"/>
+            </div>
+        </div>
+        <div class="form-group form-row" wicket:enclosure="randomOrder">
+            <div class="offset-sm-2 col-sm-10">
+                <div class="form-check">
+                    <input wicket:id="randomOrder" class="form-check-input" type="checkbox"/>
+                    <label wicket:for="randomOrder" class="form-check-label">
+                        <wicket:label key="randomOrder"/>
+                    </label>
+                </div>
+            </div>
+        </div>
+        <!-- DELETE ? -->
+        <div class="form-group form-row" wicket:enclosure="seed">
+            <label class="col-sm-2 col-form-label">
+                <wicket:message key="seed"/>
+            </label>
+            <div class="col-sm-10">
+                <input wicket:id="seed" type="number" min="0" max="1000" class="form-control"/>
```

Comment:
Min/max values should be set on the Wicket component in the Java code, not in the HTML.